build.gradle

```diff
@@ -34,8 +35,8 @@ gradlerio.team = "1294"                                   //Your FRC team number
 
 
 dependencies {
-	compile files(System.properties['user.home'] + '/wpilib/java/extracted/library/lib/NetworkTables.jar')
-	compile files(System.properties['user.home'] + '/wpilib/java/extracted/library/lib/WPILib.jar')
+    compile files(System.properties['user.home'] + '/wpilib/java/extracted/library/lib/NetworkTables.jar')
```

Comment:
You have many changes that are whitespace only. I would suggest that we come up with a whitespace standard (pick tabs or spaces) and enforce it.